Lang/English.pj.Lang

```diff
@@ -88,6 +88,7 @@
 #198#  "Slot 8"
 #199#  "Slot 9"
 #200#  "Slot 10"
+#201#  "Save Slot (%ws) selected"

```

Comment:
It doesn't look good, but I didn't want to hard code the %ws part, since the word order can be switched in different languages.